libkbfs/disk_block_cache.go

```diff
@@ -320,6 +370,27 @@ func (cache *DiskBlockCacheLocal) syncBlockCountsFromDb() error {
 	cache.numBlocks = numBlocks
 	cache.tlfSizes = tlfSizes
 	cache.currBytes = totalSize
+
+	cache.log.Debug("+ syncBlockCountsAndUnrefsFromDb block counts done")
```

Comment:
Shouldn't this be `| syncBlockCounts...`?

Reply:
I don't know what that syntax means,I was just copy-pasting.  But happy to change it if there's some meaning behind it.

Reply:
The client applies this more strictly, but `+` refers to a method starting, and `-` refers to a method ending. `|` refers to it continuing, but that's not used as strictly.